### DISCUSSION TITLE
Hide anime (japan cartoon) from  "you may also like" in show details page

### DISCUSSION THREAD

**Discussion by blackholeearth** [trakt/showly#543] [Ideas / Feature Requests]
* Hide Anime (japan cartoon) from  "you may also like" in show details page

This option can be added to options page.

**Comment by michaldrabik** [comment]
Seems very specific :)
Do you think this would be a big of an issue for majority of users to add separate option for this thing specifically?

**Reply by blackholeearth** [reply]
İ already track anime with MAL app. 

İ am watching a show . i like it very much. Then i want to find  similar shows . İn  some series. İ see its full of anime. 

 İts annoying. But app works. 

. Without putting options. You can kill animes permanently if thats easier to do.

**Reply by Fooftilly** [reply]
I could be broader. For example, hide category (western, anime...) instead of just anime. Have a checkbox list of genres in options so we can uncheck genres that we don't want to see. Could also be nice to have an option to hide a media network (e.g. TV Tokyo). I think this could be more useful for wider range of users.
